client/packages/programs/src/JsonForms/components/Search/SearchWithUserSource.tsx

```diff
@@ -100,54 +95,44 @@ export const SearchWithUserSource = (
         renderers={renderers}
         enabled={!isPatientSelected}
       />
-      {(isPatientSelected || results.length > 0) && (
-        <DetailInputWithLabelRow
```

Comment:
this diff is just remove this component wrapper, seeing as there was no label...